applications/matter_weather_station/README.rst

```diff
@@ -13,7 +13,8 @@ The device works as a Matter accessory device, meaning it can be paired and cont
 You can use this application as a reference for creating your own application.
 
 .. note::
-    The Matter protocol is in an early development stage and must be treated as an :ref:`experimental <software_maturity>` feature.
+   The Thingy:53 Matter weather station application featured in the |NCS| v2.1.1 release participated in Matter Specification Validation Event (SVE) and successfully passed all required test cases to be considered as a device compliant with Matter 1.0.
+   You can use |NCS| v2.1.1 release to see the configuration and files that were originally used in Matter 1.0 certification.
```

Comment:
Apply edits to this file as discussed in https://github.com/nrfconnect/sdk-nrf/pull/9002 from @kkasperczyk-no

Reply:
@greg-fer I believe this PR will be rebased after merging #9002 and my commit was pulled only to provide some base for testing Damian changes, so maybe modifying it is not needed.

Reply:
Ah ok, mysteries of the GitGods!

Reply:
Rebased. Please re-review. @greg-fer - yes, I guess I still need your approval although you already reviewed it on the other branch.